integration_files/SweRV_EH1/snapshots/default/common_defines_verilator.vh

```diff
@@ -0,0 +1,194 @@
+// NOTE NOTE NOTE NOTE NOTE NOTE NOTE NOTE NOTE NOTE NOTE NOTE NOTE NOTE NOTE NOTE
```

Comment:
What is this auto generated file?

Reply:
@moizhussain-lm SweRV comes with a number of configurations. Defines for each configuration are generated when used for the first time. Right now we have targetted only  the default configuration for which it is fine. I think we should add support for other configurations. I am creating an issue for that. Till then this file is fine.

Reply:
I have created a separate issue #86  for this.  Resolving it for now.